client/my-sites/domains/domain-management/edit/domain-types/transfer-in-domain-type.jsx

```diff
@@ -0,0 +1,231 @@
+			return null;
+		}
+
+		if ( isCancelable( purchase ) ) {
+			const link = cancelPurchase( selectedSite.slug, purchase.id );
+
+			return <VerticalNavItem path={ link }>{ title }</VerticalNavItem>;
+		}
+
+		return (
+			<RemovePurchase
+				hasLoadedSites={ true }
+				hasLoadedUserPurchasesFromServer={ true }
+				site={ selectedSite }
+				purchase={ purchase }
+				title={ title }
+				hideTrashIcon={ true }
+				displayButtonAsLink={ true }
+			/>
+		);
+	}
+
+	resolveStatus() {
+		const { domain, translate } = this.props;
+		const { transferStatus: status } = domain;
+
+		if ( status === transferStatus.PENDING_START ) {
+			return {
+				statusText: translate( 'Action required' ),
+				statusClass: 'status-error',
+				icon: 'info',
+			};
+		} else if ( status === transferStatus.CANCELLED ) {
+			return {
+				statusText: translate( 'Transfer failed' ),
+				statusClass: 'status-error',
+				icon: 'info',
+			};
+		}
+
+		return {
+			statusText: translate( 'Transfer in progress' ),
+			statusClass: 'status-success',
+			icon: 'cached',
+		};
+	}
+
+	startTransfer = () => {
+		const { domain, selectedSite } = this.props;
+		page( domainManagementTransferInPrecheck( selectedSite.slug, domain.name ) );
+	};
+
+	renderPendingStart() {
+		const { domain, translate } = this.props;
+
+		return (
+			<>
+				<p>
+					{ translate(
+						'We need you to complete a couple of steps before we can transfer %(domain)s from your ' +
+							'current domain provider to WordPress.com. Your domain will stay at your current provider ' +
+							'until the transfer is completed.',
+						{
+							args: {
+								domain: domain.name,
+							},
+						}
+					) }
+				</p>
+
+				<Button primary onClick={ this.startTransfer }>
+					{ translate( 'Start transfer' ) }
+				</Button>
+			</>
+		);
+	}
+
+	renderTransferInProgress() {
+		const { translate } = this.props;
+
+		return (
+			<p>
+				{ translate(
+					'Your transfer has been started and is waiting for authorization from your current ' +
+						'domain provider. This process can take up to 7 days. If you need to cancel or expedite the ' +
+						'transfer please contact them for assistance.'
+				) }
+			</p>
+		);
+	}
+
+	renderTransferFailed() {
+		const { domain, translate } = this.props;
+
+		return (
+			<>
+				<p>
+					{ translate(
+						'We were unable to complete the transfer of {{strong}}%(domain)s{{/strong}}. ' +
+							'You can remove the transfer from your account or try to start the transfer again. ' +
+							'{{a}}Learn more{{/a}}',
+						{
+							args: {
+								domain: domain.name,
+							},
+							components: {
+								strong: <strong />,
+								a: (
+									<a
+										href={ INCOMING_DOMAIN_TRANSFER_STATUSES }
+										target="_blank"
+										rel="noopener noreferrer"
+									/>
+								),
+							},
+						}
+					) }
+				</p>
+
+				<Button onClick={ this.startTransfer }>{ translate( 'Start transfer again' ) }</Button>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Start Transfer Again' )` *ES Score: 10*